src/server.js

```diff
@@ -52,6 +54,61 @@ export class Server {
     }
   }
 
+  /**
+   * Get timebounds for N seconds from now, when you're creating a transaction
+   * with {@link TransactionBuilder}.
+   *
+   * By default, {@link TransactionBuilder} uses the current local time, but
+   * your machine's local time could be different from Horizon's. This gives you
+   * more assurance that your timebounds will reflect what you want.
+   *
+   * Note that this will generate your timebounds when you **run the function**,
+   * not when you submit the function! So give yourself enough time to get
```

Comment:
Shouldn't be "submit the ~function~ transaction"?

Reply:
NIce catch!